reference/best_practices/reusable_bundles.rst

```diff
@@ -1,4 +1,95 @@
 Reusable Bundles for Akeneo PIM community
 =========================================
 
-Coming soon!!! :)
+You're integrator and you want to share your project with the awesome Akeneo PIM community?
+Let's read our advices to make a reusable and documented bundle!
+
+
+What do you want when you are creating a reusable bundle?
+
+* **Compatibility**: fit Akeneo PIM features
+* **Maintainability**: make your code flexible and testable (and tested)
+* **Extensibility**: to allow integrators to fit your bundle with their specific needs
+* **Scalability**: know your application limits and document them
+
+
+Compatibility
+-------------
+
+Your bundle won't be reusable for some customers if you forgot some cases. But that's not a problem if they are aware of the compatibility of your bundle.
+
+Akeneo PIM storage stack comes with MySQL and hybrid MySQL/MongoDB (:doc:`/reference/technical_information/choose_database`).
+Use the ProductQueryBuilder which is an overlayer making work both storages with the same API.
```

Comment:
@rybus, @LaureBro I rewrite it like this: "Use the ProductQueryBuilder which provides an abstraction layer making both storages work with the same API."
Is it ok for you?